app/models/content.rb

```diff
@@ -18,7 +18,7 @@ def update_attributes_with_content(content)
       self.try("#{at}=", content[at])
     end
 
-    self.original_url = content['url']
+    self.href = content['url']
```

Comment:
So if i'm following this correctly ... this is basically doing the same thing, except that it's going to null-out the status every time the original_url changes.  Which will cause the representer to return the original url instead of the completed/processed url.

Right?